### PR TITLE
title encoding issue fixed.

### PR DESCRIPTION
**Pull request by Kidsunbo**
When the title parameter is of the type "str", it will be encoded to bytes. Otherwise, just forward the parameter.

### CODE REVIEW

File: glfw.py [hunk 1]
```diff
@@ -480,7 +480,7 @@ class GLFWmonitor(Structure): pass
 def glfwCreateWindow(width=640, height=480, title="GLFW Window",
                      monitor=None, share=None):
     _glfw.glfwCreateWindow.restype = POINTER(GLFWwindow)
-    window = _glfw.glfwCreateWindow(width,height,title,monitor,share)
+    window = _glfw.glfwCreateWindow(width,height,title if type(title) == bytes else title.encode('utf-8'),monitor,share)
```
Comment (rougier) [inline glfw.py]: Maybe you can add a dedicated line for title transformation (more readable I think)

**Comment by rougier** [comment]
Thanks ! See my comment. Also, the set_title function might need some fix too.

File: glfw.py (hunk 1 above)
Comment (Kidsunbo) [reply glfw.py]: > Maybe you can add a dedicated line for title transformation (more readable I think)

Yeah, sure. A dedicated line could be better. I have changed the code. The set_title function has also be fixed.

**Comment by rougier** [comment]
Thanks !

**Pull request merged**